### PR TITLE
add update functionality to people writing

### PR DESCRIPTION
**Pull request by AdamDM**

### CODE REVIEW

File: src/main/java/org/atlasapi/persistence/content/mongo/MongoPersonStore.java [hunk 1]
```diff
@@ -101,6 +101,17 @@ public void createOrUpdatePerson(Person person) {
         lookupEntryStore.store(LookupEntry.lookupEntryFrom(person));
         writeEquivalences(person);
     }
+
+    public void updatePerson(Person person) {
```
Comment (jamiesprax) [inline src/main/java/org/atlasapi/persistence/content/mongo/MongoPersonStore.java]: Override annotation?

File: src/test/java/org/atlasapi/persistence/media/entity/PersonTranslatorTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        DBObject dbObject = personTranslator.toDBObject(new BasicDBObject(), person);
+        personTranslator.writeUpdate(dbObject, newPerson);
+        Person resultPerson = personTranslator.fromDBObject(dbObject, new Person());
+
+        assertEquals(newPerson.getFamilyName(), resultPerson.getFamilyName());
+        assertEquals(newPerson.getGivenName(), resultPerson.getGivenName());
+        assertEquals(person.getGender(), resultPerson.getGender());
+
+    }
+}
```
Comment (dragospitica) [inline src/test/java/org/atlasapi/persistence/media/entity/PersonTranslatorTest.java]: no new line?

File: src/main/java/org/atlasapi/persistence/media/entity/PersonTranslator.java [hunk 3]
```diff
@@ -48,6 +48,17 @@ public Person fromDBObject(DBObject dbObject, Person entity) {
         return entity;
     }
 
+    public DBObject writeUpdate(DBObject dbObject, Person entity) {
```
Comment (dragospitica) [inline src/main/java/org/atlasapi/persistence/media/entity/PersonTranslator.java]: this needs a better name as it's not writing anything

File: src/main/java/org/atlasapi/persistence/content/people/QueuingPersonWriter.java [hunk 4]
```diff
@@ -71,6 +71,8 @@ public void run() {
                     if (! processedUris.contains(person.getCanonicalUri())) {
                         if (! store.person(person.getCanonicalUri()).isPresent()) {
                             store.createOrUpdatePerson(person);
+                        } else {
+                            store.updatePerson(person);
```
Comment (jamiesprax) [inline src/main/java/org/atlasapi/persistence/content/people/QueuingPersonWriter.java]: This looks like createOrUpdatePerson must not be doing any updating? It seem's its only called if the Person is not present. If it does have update logic, surely the check could be removed and then the Person would be updated through there?

Comment (dragospitica) [inline src/main/java/org/atlasapi/persistence/content/people/QueuingPersonWriter.java]: following the naming of the methods, the method above should only create a person

**Comment by jamiesprax** [comment]
Your implementation of this has a lot of code duplication that looks it can be avoided:
* createOrUpdatePerson() and UpdatePerson()
* writeUpdate() and toDbObject()

**Pull request closed without merging**